packages/quill-next-vue/package.json

```diff
@@ -0,0 +1,48 @@
+{
+  "name": "quill-next-vue",
+  "version": "1.0.0",
+  "author": "Sannnekk <https://github.com/sannnekk>",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/vincentdchan/quill-next",
+    "directory": "packages/quill-next-vue"
+  },
+  "type": "module",
+  "main": "./dist/index.js",
+  "module": "./dist/index.js",
+  "types": "./dist/index.d.ts",
+  "exports": {
+    ".": {
+      "types": "./dist/index.d.ts",
+      "import": "./dist/index.js"
+    }
+  },
+  "scripts": {
```

Comment:
Could you add a `dev` command so I could test the preview page in the `dev` dir?